src/components/tabs/TabNavigation.tsx

```diff
@@ -79,7 +81,7 @@ const TabNavigation = ({
       centerContent={true}
       showsHorizontalScrollIndicator={false}
       contentContainerStyle={[
-        styles.container,
+        excludePadding ? styles.containerNoPadding : styles.container,
```

Comment:
If we change the naming, we can also make the logic more explicit by using a condition without negation:
```suggestion
        includeContentMargins
          ? { paddingHorizontal: IOVisualCostants.appMarginDefault }
          : {},
```
We should also add `flexGrow` in the following style:
```jsx
{
  flexGrow: 1,
  justifyContent: itemsJustify[tabAlignment]
}
```